bigbone/src/main/kotlin/social/bigbone/api/entity/Instance.kt

```diff
@@ -85,9 +85,214 @@ data class Instance(
     @SerializedName("invites_enabled")
     val invitesEnabled: Boolean = false,
 
+    /**
+     * Configured values and limits for this website.
+     */
+    @SerializedName("configuration")
+    val configuration: Configuration? = null,
+
     /**
      * A user that can be contacted, as an alternative to email.
      */
     @SerializedName("contact_account")
-    val contactAccount: Account? = null
-)
+    val contactAccount: Account? = null,
+
+    // Add rules block
+    /**
+     * An itemized list of rules for this website.
+     */
+    @SerializedName("rules")
+    val rules: List<Rule>? = null
+) {
+    /**
+     * URLs of interest for clients apps.
+     * @see <a href="https://docs.joinmastodon.org/entities/V1_Instance/">Mastodon API V1::Instance</a>
+     */
+    data class InstanceUrls(
```

Comment:
As a nested class, could be renamed to just `Urls`.

Reply:
Thx, fixed!